runner.go

```diff
@@ -114,6 +116,9 @@ func Run(options ...RunOption) {
 	config.Watch = !nonInteractive
 	if len(runtime.flags.Args()) > 0 {
 		config.Watch = false
+		if interactive {
+			cofig.Watch = true
```

Comment:
this series of booleans flow in a way that's hard to reason about to me

Reply:
I didn't want to change the current behavior of defaulting to false and confuse people.

Reply:
i don't think anyone uses non-interactive (highly doubt anyone even knows it exists)